.circleci/config.yml

```diff
@@ -46,17 +46,6 @@ jobs:
               source activate mne
               echo "source activate mne" >> $BASH_ENV;
 
-        # The conda-provided PyQt5 does not work on CircleCI for some reason,
-        # possibly because the libGL expects a sufficiently modern libgcc
-        # and conda rolls their own (libgcc-ng package).
-        - run:
-            name: Patch Qt
-            command: |
-              conda remove --yes qt pyqt matplotlib sip libxcb icu vtk
-              pip uninstall --yes mayavi vtk
-              pip install vtk mayavi PyQt5 PyQt5-sip sip matplotlib
-              echo "export LD_PRELOAD=~/miniconda/envs/mne/lib/libgobject-2.0.so.0" >> $BASH_ENV
-              pip install --upgrade "https://api.github.com/repos/nipy/PySurfer/zipball/master"
```

Comment:
You will need mayavi for the report script

Reply:
mayavi should be there already. But yes I need to understand why it was not passing.